proto/backup.proto

```diff
@@ -72,6 +72,10 @@ message Schema {
     uint64 total_bytes = 5;
 
     uint32 tiflash_replicas = 6;
+
+    // json_table represents dump stats for a table, which generate by DumpStatsToJSON
+    // https://github.com/pingcap/tidb/blob/e136429d8dc5d70f43cd3f94179b0b9f47595097/statistics/handle/dump.go#L116
+    bytes json_table = 7;
```

Comment:
JSONTable is just the name of the type. The name of the variable should be "stats".

Reply:
Done, PTAL~